src/auth/login-form/login-form.js

```diff
@@ -55,28 +57,34 @@ class LoginForm extends Component {
       <div className="login-details">
         <Form>
           <Form.Group controlId="formBasicEmail">
-            <Form.Label>Email address</Form.Label>
-            <Form.Control
-              required
-              type="email"
-              placeholder="abc@gmail.com"
-              name="email"
-              onChange={event => this.handleChange(event, "email")}
-            />
-            <Form.Text className="text-muted">
-              We'll never share your email with anyone else.
-            </Form.Text>
+          <TextField
+          id="outlined-full-width"
+          label="Email"
+          style={{ margin: 8 }}
+          placeholder="abc@donut.com"
+          helperText="We will not share your information with anyone"
+          fullWidth
+          margin="normal"
+          InputLabelProps={{
+            shrink: true,
+          }}
+          variant="outlined"
+        />
           </Form.Group>
 
           <Form.Group controlId="formBasicPassword">
-            <Form.Label>Password</Form.Label>
-            <Form.Control
-              required
-              type="password"
-              placeholder="***********"
-              name="password"
-              onChange={event => this.handleChange(event, "password")}
-            />
+          <TextField
+          id="outlined-full-width"
+          label="Password"
+          style={{ margin: 8 }}
```

Comment:
I think your ESLint setup has caused some problems with the indentation. Please correct that. The closing tag and attributes aren't properly indented.

Reply:
Can you see it now.